src/views/ResetPassword.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <section id="login">
```

Comment:
you shouldn't use id login here ! for many reasons ! CSS and cypress and the whole purpose of using `id` to target unique elements.